assets/pages/error/components/errorPage.jsx

```diff
@@ -82,3 +83,7 @@ function ReportLink(props: { show: boolean }) {
 ErrorPage.defaultProps = {
   reportLink: false,
 };
+
+ReportLink.defaultProps = {
+  countryGroupId: 'GBPCountries',
+};
```

Comment:
Since `countryGroupId` is not an optional prop, I think it is not necessary to provide a default value for it.

Reply:
This is where Flow and ESLint conflict (see #702). We've gone with Flow for this, and Flow requires you to mark default props as required; it picks up the fact that they're optional from your `defaultProps` declaration.

Reply:
done